src/Chainweb/Chainweb.hs

```diff
@@ -611,6 +633,7 @@ withChainwebInternal conf logger peer rocksDb dbDir nodeid resetDb inner = do
                             , _chainwebThrottler = throttler
                             , _chainwebMiningThrottler = miningThrottler
                             , _chainwebPutPeerThrottler = putPeerThrottler
+                            , _chainwebLocalThrottler = localThrottler
```

Comment:
Are we actually serving this middleware?

Reply:
I'll doublecheck.

Reply:
It must be added to this:

```Haskell
IrunChainweb cw = do
    logg Info "start chainweb node"
    concurrently_
        -- 1. Start serving Rest API
        (serve
            $ httpLog
            . _chainwebRouteBlacklist cw
            . throttle (_chainwebPutPeerThrottler cw)
            . throttle (_chainwebMiningThrottler cw)
            . throttle (_chainwebThrottler cw)
        )
```